components/menu/menu-divider.directive.ts

```diff
@@ -3,14 +3,15 @@
  * found in the LICENSE file at https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/LICENSE
  */
 
-import { Directive, ElementRef, Renderer2 } from '@angular/core';
+import { Directive, ElementRef } from '@angular/core';
 
 @Directive({
   selector: '[nz-menu-divider]',
-  exportAs: 'nzMenuDivider'
+  exportAs: 'nzMenuDivider',
+  host: {
+    class: 'ant-dropdown-menu-item-divider'
+  }
 })
 export class NzMenuDividerDirective {
-  constructor(public elementRef: ElementRef, private renderer: Renderer2) {
-    this.renderer.addClass(elementRef.nativeElement, 'ant-dropdown-menu-item-divider');
-  }
+  constructor(public elementRef: ElementRef) {}
```

Comment:
This `ElementRef` is not used inside the component, but since it is public, I did not remove it. 
Maybe I should mark it as deprecated?